### PR TITLE
Towardsagent3

### PR DESCRIPTION
**Pull request by ntartania**
agent 3 moving randomly and pooping (I know, it's a bit much for a single pull request).

@pgavigan 
@esfandia

### CODE REVIEW

**Comment by ntartania** [comment]
adresses issues #2 and #3

**Comment by esfandia** [comment]
Works for me (but only after I re-enable repaint() ;) )

File: r3.asl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+!explore(slots).
+
+/* Plans */
+
++!explore(slots) : true
+   <- randMove(slot);
+   	  maybePoop(garb);
+      !explore(slots).
+	  
+//+!randmove(slot): true
```
Comment (esfandia) [inline r3.asl]: might want to delete commented code

Comment (pgavigan) [reply r3.asl]: Is your approval to merge contingent on deleting commented code? Is this pull request approved?

**Pull request merged**